### ISSUE TITLE
asdf-mage is not in the official asdf-plugins

### ISSUE THREAD

**Issue by abatilo** [ggilmore/asdf-mage#1]
This means that the `asdf plugin add mage` command that's listed in the README is not enough to install this plugin. You have to specify the plugin location.

Could we get this plugin added to the main repository?

**Comment by ggilmore** [comment]
I can do that, but it'll take some time before they accept the PR (in my experience)

**Comment by abatilo** [comment]
Yeah, I've noticed that too. I'd appreciate it though if we could start that process!

**Comment by mathew-fleisch** [comment]
@ggilmore beat you to it 😏 
https://github.com/mathew-fleisch/asdf-mage
https://github.com/asdf-vm/asdf-plugins/pull/507
@abatilo fyi
https://i.imgur.com/WB0l53y.gif

**Comment by chrisgoffinet** [comment]
@abatilo is this enough to close this out now that it's listed?
